fakes/version_parser.go

```diff
@@ -4,7 +4,7 @@ import "sync"
 
 type VersionParser struct {
 	ParseVersionCall struct {
-		mutex     sync.Mutex
+		sync.Mutex
```

Comment:
Can you upgrade your version of `faux` and regenerate these fakes so that we don't get these reversions of the explicit mutex named field?

Reply:
done in https://github.com/paketo-buildpacks/mri/pull/321/commits/1b2aba1790453afe56544ce97444344dac8da168